festipals/src/css/index.js

```diff
@@ -0,0 +1,54 @@
+import React, {StyleSheet, Dimensions, PixelRatio} from "react-native";
+const {width, height, scale} = Dimensions.get("window"),
+    vw = width / 100,
+    vh = height / 100,
+    vmin = Math.min(vw, vh),
+    vmax = Math.max(vw, vh);
+
+export default StyleSheet.create({
+    "body": {
+        "marginTop": 0,
+        "marginRight": 0,
+        "marginBottom": 0,
+        "marginLeft": 0,
```

Comment:
Instead of writing everything seperately, just write `margin: 0 0 0 0;` instead.